dbt/compilation.py

```diff
@@ -132,7 +132,7 @@ def compile_node(self, node, manifest, extra_context=None):
 
         injected_node, _ = prepend_ctes(compiled_node, manifest)
 
-        should_wrap = {NodeType.Test, NodeType.Analysis, NodeType.Operation}
+        should_wrap = {NodeType.Test, NodeType.Operation}
```

Comment:
i'm unsure why `Analysis` nodes were ever in here. We definitely do not wrap them. This whole function is... not good. Would be great to revisit at some point soon